src/client/App.js

```diff
@@ -1,17 +1,58 @@
 import React, { useEffect, useState } from 'react';
 import { BrowserRouter as Router, Switch, Route } from 'react-router-dom';
-import { Login, Registro, Home } from './pages/';
-import usuarioContexto from './state/state';
+import { decodeToken } from './helpers/jwt';
+import { Login, Register, Home } from './pages/';
+import authContext from './state/state';
+// import usuarioContexto from './state/state';
 
 export default function App() {
-  const [user, setUser] = useState({ nombre: '', uid: '', islogged: false });
+  const [user, setUser] = useState({ islogged: false });
```

Comment:
What's the purpose of the `islogged` property? Why couldn't `user` be `null` instead?